std/split/split_gl.asm

```diff
@@ -1,27 +1,25 @@
 // Splits an arbitrary field element into two u32s, on the Goldilocks field.
-machine SplitGL(RESET, operation_id) {
+machine SplitGL(RESET, _) {
 
-    operation split<0> in_acc -> output_low, output_high;
+    operation split in_acc -> output_low, output_high;
 
     // Latch and operation ID
     col fixed RESET(i) { i % 8 == 7 };
-    col witness operation_id;
 
     // 1. Decompose the input into bytes
 
     // The byte decomposition of the input, in little-endian order
     // and shifted forward by one (to use the last row of the
     // previous block)
-    col witness bytes;
-    // Puts the bytes together to form the input
+    col witness bytes(i) query ("hint", (in_acc' >> (((i + 1) % 8) * 8)) % 0xff);
+    // Puts the bytes together to from the input
```

Comment:
Oh LOL, I should undo that :D

Reply:
was this GPT generated? :eyes:

Reply:
No, I thought I found a typo, but it wasn't. I think GPT would've gotten that!

Reply:
I apologize for the oversight. You are correct, and I made an error in my response.

Reply:
Most of the repetitive part in this file is generated by copilot though 😉